### ISSUE TITLE
Make regression issues queryable in issue platform

### ISSUE THREAD

**Issue by cathteng** [getsentry/sentry#71031]
### Problem Statement

We would like to add function regression issues (and possibly other regression issues) to open PR comments.

Currently, the issue platform (the `search_issues` dataset in Snuba) supports querying for normal issues, but regression issues also have information in the occurrence that would be useful for querying like the file and function, and other information that would be useful to know about like the subtitle (e.g. `Duration increased from 510.82ms to 791.56ms (P95)`). These fields are currently stored in the `occurrence`, which is stored in nodestore.

TL;DR The issue platform has an `occurrence_id` field that points to the occurrence, but could we pull out some fields from the occurrence itself and store it directly in the table?

There may also be other use cases for this !

### Product Area

Issues

**Comment by mrduncan** [comment]
@cathteng can you store the data you want to query on in the evidence?

cc @JoshFerge
